appveyor.yml

```diff
@@ -16,9 +18,15 @@ before_build:
         If ($env:Platform -Match "x86") {
             $env:VCVARS_PLATFORM="x86"
             $env:TARGET="VC-WIN32 no-asm"
-        } Else {
+        } Else If ($env:Platform -Match "x64") {
             $env:VCVARS_PLATFORM="amd64"
             $env:TARGET="VC-WIN64A-masm"
+        } Else If ($env:Platform -Match "arm") {
+            $env:VCVARS_PLATFORM="amd64_arm"
+            $env:TARGET="VC-WIN32-ARM"
```

Comment:
It's `elseif` in powershell-ska, not `else if`. However! What is expectation here? That appveyor has ARM compilers, and will even execute ARM binary code? Well, it might have compilers, but I doubt that it would execute binary code. But as far as I understand ARM compiler availability is complex function of Visual Studio version and maybe even edition. For example I don't have no ARM64 compiler in my VS2017 Community installation... Is it established that appveyor has it for `VSVER: 14` that we have in appveyor.yml? And I keep forgetting what `VSVER: 14` mean. I can't see it documented on https://www.appveyor.com/docs/...

Reply:
> ARM compiler availability is complex function of Visual Studio version and maybe even edition.

... and installation options. Basically I fail to see that it's actually sufficient to say "just run vcvarsall.bat with this-n-that argument". For starters vcvarsall doesn't seem to be anywhere on the %PATH%, you have to look for it. All this is to the question what kind of documentation that is needed. However! I don't mean step-by-step, general directions should do, but at least one shouldn't assume that it's just a common knowledge. And since suggestion is to define targets in separate .conf file, this general directions should be described directly there.

Reply:
People are already expected to call vcvarsall.bat for VC-WIN32 and VC-WIN64A/I, so that didn't change. Fair point about having to select the compiler for installation.

Reply:
> People are already expected to call vcvarsall.bat

OK. What do I know, as I never did, I just choose suitable Tools Command Prompt... Since forever... But sure, if majority is accustomed to calling vcvarsall.bat, then fine. Here is an observation by the way. appveyor.yml says that it resides in  `%VSCOMNTOOLS%\..\..\VC\`. I don't seem to have it in my VS2017 Community, and vcvarsall.bat resides in `VC\Auxiliary\Build`... What I'm trying to say is that it doesn't necessarily work to assume that everybody is on exactly same page :-)

Reply:
Ah, good point. If the ARM/ARM64 tools are installed, there should be "Command Prompt" entries in the start menu as well, because the menu entries directly correspond to vcvarsall invocations.

Reply:
> there should be "Command Prompt"

Do *you* have them? As I don't... I only have x86 and x64. What's your version anyway? And not only number but even edition?